src/components/movieDetailsPage/MovieDetailsPage.test.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import MovieDetailsPage from './MovieDetailsPage';
+
+test('render MovieDetailsPage', () => {
+  const { getByTestId } = render(<MovieDetailsPage />);
+  const logoEl = getByTestId('logo');
+  expect(logoEl.textContent).toBe('FILMS');
+});
```

Comment:
Try to check a conditionsion the test `{isAboutFilmActive ? ... : ... }`
And some clicks on the buttons